docs/topics/http/sessions.txt

```diff
@@ -676,6 +677,125 @@ Technical details
 * Django only sends a cookie if it needs to. If you don't set any session
   data, it won't send a session cookie.
 
+.. _extending-database-backed-session-engines:
+
+Extending database-backed session engines
+=========================================
+
+.. versionadded:: 1.9
+
+Creating a custom database-backed session engine built upon those included 
+in Django (namely ``db`` and ``cached_db``) may be done via the inheritance
+of the abstract session model :class:`models.AbstractBaseSession`, and an either
+one of ``SessionStore`` classes.
```

Comment:
Added a section documenting base classes to inherit from:
`Base classes for building SessionStore objects are <...>`